src/scripts/components/Login.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+import LoginFormContainer from 'containers/LoginFormContainer'
+
+const Wrapper = styled.div`
+  font-size: 20px;
+`
+
+const Title = styled.header`
+  background-color: #fff;
+  height: 50px;
+  padding: 20px;
+  display: flex;
+  align-items: center;
+  color: #95989a;
```

Comment:
I think we should move all of our color values to a new `constants/StyleConstants.js` file, and re-create the color constants we had in sass in `paratii-player`. I'll create an issue for this